picker-starter/src/picker.config.ts

```diff
@@ -0,0 +1,40 @@
+import { getProductFilters, queryCategories, queryProducts } from '@/data'
+import { defineConfig } from '@/core'
+import { StoryblokIcon } from './components'
+import { optionsSchema } from './options-schema'
+
+export default defineConfig((options) => {
+  return {
+    title: 'Picker Starter',
+    icon: StoryblokIcon,
+    options: {
+      example: {
+        limit: '10',
+        selectOnly: 'product',
+      },
+      validate: () => {
+        const validationResult = optionsSchema.validate(options, {
+          abortEarly: false,
+        })
+
+        return {
+          isValid: !validationResult.error,
+          error: validationResult.error?.message,
+        }
+      },
+    },
+    tabs: [
+      {
+        name: 'product',
+        label: 'Products',
+        query: queryProducts,
```

Comment:
In another PR we may place all the `queryProducts` logic directly here.

Reply:
Thanks for this reminder, @eunjae-lee. 💯 

I had created two tasks for handling these points. 🙌